site/_contributors/addy.txt

```diff
@@ -0,0 +1,6 @@
+---
+weight: 1
+name: Addy
+image: /assets/images/addy.png
```

Comment:
Compresser.io wasn't giving much wins for these images, so just searched around and replaced them with much smaller images.